client/web/views/root_view/add_access_key_dialog/index.ts

```diff
@@ -165,7 +165,7 @@ export class AddAccessKeyDialog extends LitElement {
   }
 
   private async runAccessKeyChecks(accessKey: string | null) {
-    if (accessKey === null) {
+    if (accessKey === null || accessKey === '') {
```

Comment:
I don't think this change is related to the main goal. We only need to figure out why the events are not reaching `rootEl`. Please remove this.